src/bonuses.cpp

```diff
@@ -0,0 +1,275 @@
+#include "bonuses.h"
+#include "damage.h"
+#include "json.h"
+#include "character.h"
+#include "debug.h"
+#include <map>
+#include <string>
+#include <utility>
+#include <sstream>
+#include <vector>
+
+#define dbg(x) DebugLog((DebugLevel)(x),D_MAIN) << __FILE__ << ":" << __LINE__ << ": "
+
+// Tokenize
```

Comment:
> The thing has already been run through a json tokenizer, why are you adding another one?

Didn't know we have a json tokenizer.
How to get tokens from it? I can't find anything in json.h